.github/workflows/test-check-transformers.yaml

```diff
@@ -12,33 +12,28 @@ env:
 jobs:
   detect-changes:
     runs-on: ubuntu-latest
-
-    outputs:
-      changes-present: ${{ steps.changed-files.outputs.any_modified }}
-
     steps:
       - name: Checkout
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
       - name: Get changed files
         id: changed-files
-        uses: tj-actions/changed-files@v45
+        uses: neuralmagic/nm-actions/actions/changed-files@main
         with:
-          files: |
+          include-patterns: |
             **
-            !examples/**
-            !tests/e2e/**
-            !tests/lmeval/**
-            !tests/examples/**
-            !**/*.md
-            !.github/**
             .github/workflows/test-check-transformers.yaml
-
+          exclude-patterns: |
+            examples/**
+            tests/e2e/**
+            tests/lmeval/**
+            tests/examples/**
+            **/*.md
+            .github/**
```

Comment:
Based on the logic of how the new action works, I think this input/line specifically (`.github/**`) should be removed as it's no longer necessary and would actually prevent the one included in that file from being included, since exclusions always take precedence over inclusions.